apps/src/templates/sectionProgressV2/MetadataHelpers.jsx

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+
+export function formatTimeSpent(studentProgress) {
+  if (studentProgress?.timeSpent) {
+    const minutes = studentProgress.timeSpent / 60;
+    return `${Math.ceil(minutes)}`;
+  }
+  return missingDataFormatter(!!studentProgress);
+}
+
+export function formatLastUpdated(studentProgress) {
+  if (studentProgress?.lastTimestamp) {
+    return moment.unix(studentProgress.lastTimestamp).format('M/D');
```

Comment:
The `moment` library has been deprecated for years, probably best not to use it in new code.

```suggestion
    const date = new Date(studentProgress.lastTimestamp * 1000);
    return new Intl.DateTimeFormat('en-US', {
      month: 'numeric',
      day: 'numeric'
    }).format(date);
```